### PR TITLE
Update to 0.14.0 beta4

### PR DESCRIPTION
**Pull request by maurolacy**

### CODE REVIEW

File: contracts/cw1-subkeys/src/contract.rs [hunk 1]
```diff
@@ -147,11 +147,6 @@ pub fn check_staking_permissions(
                 return Err(ContractError::ReDelegatePerm {});
             }
         }
-        StakingMsg::Withdraw { .. } => {
```
Comment (ethanfrey) [inline contracts/cw1-subkeys/src/contract.rs:150]: Hmmm... this withdraw permission just disappears?
I would add the support for the distribution messages that replace it

File: contracts/cw1-subkeys/src/contract.rs [hunk 2]
```diff
@@ -1497,13 +1475,6 @@ mod tests {
         )
         .unwrap();
         assert_eq!(res.can_execute, true);
-        let res = query_can_execute(
```
Comment (ethanfrey) [inline contracts/cw1-subkeys/src/contract.rs:1500]: The new `Distribution::Withdraw` method should be supported in this high level interface. (People think of it as staking). Not sure how you want to divide the internal processing, but this API should remain similar

File: contracts/cw20-staking/src/contract.rs [hunk 3]
```diff
@@ -356,9 +356,8 @@ pub fn reinvest(deps: DepsMut, env: Env, _info: MessageInfo) -> Result<Response,
     let res = Response {
         submessages: vec![],
         messages: vec![
-            StakingMsg::Withdraw {
+            DistributionMsg::WithdrawDelegatorReward {
```
Comment (ethanfrey) [inline contracts/cw20-staking/src/contract.rs:359]: :+1:

**Review by ethanfrey — COMMENTED** [review]
Generally good.
Let's ensure the cw1-subkeys still allows withdrawls

File: contracts/cw1-subkeys/src/contract.rs (hunk 1 above)
Comment (maurolacy) [reply contracts/cw1-subkeys/src/contract.rs:150]: FIxed.

**Review by ethanfrey — APPROVED** [review]
Nice fixes. Thank you

**Pull request merged**